python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -167,6 +170,8 @@ def setDefaults(self):
         # DiaSource Detection
         self.detection.thresholdPolarity = "both"
         self.detection.thresholdValue = 5.5
+        if self.doDecorrelation:
+            self.detection.thresholdValue = 5.0
```

Comment:
Please add a comment explaining why you change this value.

Also, since you only run decorrelation if doSubtract is True, I suggest you apply the same test here. That makes the "doCorrelate" config parameter a bit easier to explain (e.g. "ignored if doSubtract False")